docs/codeql/codeql-cli/about-codeql-packs.rst

```diff
@@ -63,41 +59,46 @@ The following properties are supported in ``qlpack.yml`` files.
    * - ``extractor``
      - ``javascript``
      - All test packs
-     - The CodeQL language extractor to use when the CLI creates a database from test files in the pack. For more information about testing queries, see ":doc:`Testing custom queries <testing-custom-queries>`."
+     - The CodeQL language extractor to use when the CLI creates a database in the pack. For more information about testing queries, see ":doc:`Testing custom queries <testing-custom-queries>`."
    * - ``tests``
      - ``.``
      - Optional for test packs
-     - Supported from release 2.1.0 onwards. The path to a directory within the pack that contains tests, defined relative to the pack directory. Use ``.`` to specify the whole pack. Any queries in this directory are run as tests when ``test run`` is run with the ``--strict-test-discovery`` option. These queries are ignored by query suite definitions that use ``queries`` or ``qlpack``    instructions to ask for all queries in a particular pack.
+     - The path to a directory within the pack that contains tests, defined relative to the pack directory. Use ``.`` to specify the whole pack. Any queries in this directory are run as tests when ``test run`` is run with the ``--strict-test-discovery`` option. These queries are ignored by query suite definitions that use ``queries`` or ``qlpack``    instructions to ask for all queries in a particular pack.
    * - ``dbscheme``
      - ``semmlecode.python.dbscheme``
-     - Core language pack only
+     - Core language packs only
      - The path to the :ref:`database schema <codeql-database-schema>` for all libraries and queries written for this CodeQL language (see example below).
    * - ``upgrades``
      - ``.``
      - Packs with upgrades
      - The path to a directory within the pack that contains upgrade scripts, defined relative to the pack directory. The ``database upgrade`` action uses these scripts to update databases that were created by an older version of an extractor so they're compatible with the current extractor (see `Upgrade scripts for a language <#upgrade-scripts-for-a-language>`__ below.)
-   * - ``collaborators``
+   * - ``authors``
      - ``example@github.com``
+     - All packs
      - Metadata that will be displayed on the packaging search page in the packages section of the organization that a CodeQL pack is published to.
-   * - ``bugs``
-     - ``https://github.com/my-org/my-repo/issues/X``
+   * - ``licenses``
+     - ``(LGPL-2.1 AND MIT)``
+     - All packs
+     - Metadata that will be displayed on the packaging search page in the packages section of the organization that a CodeQL pack is published to.
```

Comment:
Maybe link to https://spdx.org/licenses/ for a canonical list of allowable licenses?